### ISSUE TITLE
Fixing deprecation warnings

### ISSUE THREAD

**Issue by zr-zhou0o0** [sbarratt/inception-score-pytorch#17]
I encountered two warnings during calculating inception score, and I'd like to propose these fixes:

Line 37, inception_model = inception_v3(pretrained=True, transform_input=False).type(dtype)
change the pretrained=True to weights=Inception_V3_Weights.IMAGENET1K_V1

Line 44, return F.softmax(x).data.cpu().numpy()
change the softmax(x) to softmax(x, dim=1)
